src/store/modules/DownloadModule.ts

```diff
@@ -121,3 +116,16 @@ export const DownloadModule = {
         }
     },
 }
+
+function indexOfDownloadProgress(allDownloads: DownloadProgress[], assignId: number): number {
+    const index = [...allDownloads].findIndex((downloadProgress: DownloadProgress) => {
+        return downloadProgress.assignId === assignId;
+    });
+
+    if (index === -1) {
+        // The DownloadProgress by the ID wasn't found at all.
+        console.warn(`Couldn\'t find DownloadProgress object with assignId ${assignId}.`);
```

Comment:
Nitpick: no need to escape the apostrophe when using backticks to define a string, just makes the reading harder.